### PR TITLE
Content: Flight suit and helmet

### PR DESCRIPTION
**Pull request by faefux**
#### Summary
Content "Flight suit and helmet"

#### Purpose of change

Just more clothing, heli pilot equipment, namely for: #49114

#### Describe the solution

Added a flight helmet and suit. 
Jupsuit information came from: https://carterny.com/product/nomex-flight-suit-us-military-cwu-27p/
The helmet took more guess work and basing on another helmet, this should have some more eyes look over it certainly.

#### Describe alternatives you've considered

Just using generic helmet and jumpsuit. Seemed less interesting though.

#### Testing

Tested on my end and worked fine but also had a lot of problems with git so I may have broke it.

#### Additional context

Non.

### CODE REVIEW

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -3152,5 +3152,110 @@
...
     },
     "delete": { "flags": [ "DEAF" ] }
+  },
+  {
+    "id": "flight_helmet",
+    "repairs_like": "nomex_hood",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "military flight helmet" },
+    "description": "A military flight helmet. Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitting with plastic rails to allow for different load-outs.",
```
Comment (BrettDong) [inline data/json/items/tool_armor.json:3162]:
```suggestion
    "description": "A military flight helmet.  Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitting with plastic rails to allow for different load-outs.",
```

File: data/json/items/tool_armor.json [hunk 2]
```diff
@@ -3152,5 +3152,110 @@
...
+      }
+    ]
+  },
+  {
+    "id": "flight_helmet_on",
+    "repairs_like": "nomex_hood",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "military flight helmet" },
+    "description": "A military flight helmet. Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitting with plastic rails to allow for different load-outs.",
```
Comment (BrettDong) [inline data/json/items/tool_armor.json:3217]:
```suggestion
    "description": "A military flight helmet.  Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitting with plastic rails to allow for different load-outs.",
```

File: data/json/items/tool_armor.json [hunk 3]
```diff
@@ -3152,5 +3152,110 @@
...
+    "flags": [ "VARSIZE", "STURDY", "DEAF", "SUN_GLASSES" ],
+    "charges_per_use": 1,
+    "ammo": "battery",
+    "use_action": {
+      "type": "transform",
+      "msg": "You turn the flight helmet on.",
+      "target": "flight_helmet_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The helmets batteries are dead."
```
Comment (Night-Pryanik) [inline data/json/items/tool_armor.json:3190]:
```suggestion
      "need_charges_msg": "The helmet's batteries are dead."
```

File: data/json/items/tool_armor.json (hunk 2 above)
Comment (faefux) [reply data/json/items/tool_armor.json:3217]: cheers bruv.

File: data/json/items/tool_armor.json [hunk 4]
```diff
@@ -3152,5 +3152,110 @@
...
     },
     "delete": { "flags": [ "DEAF" ] }
+  },
+  {
+    "id": "flight_helmet",
+    "repairs_like": "nomex_hood",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "military flight helmet" },
+    "description": "A military flight helmet.  Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitting with plastic rails to allow for different load-outs.",
```
Comment (chaohedgehogs) [inline data/json/items/tool_armor.json:3162]:
```suggestion
    "description": "A military flight helmet.  Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitted with plastic rails to allow for different load-outs.",
```

File: data/json/items/tool_armor.json [hunk 5]
```diff
@@ -3152,5 +3152,110 @@
...
+      }
+    ]
+  },
+  {
+    "id": "flight_helmet_on",
+    "repairs_like": "nomex_hood",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "military flight helmet" },
+    "description": "A military flight helmet. Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitting with plastic rails to allow for different load-outs.",
```
Comment (chaohedgehogs) [inline data/json/items/tool_armor.json:3217]:
```suggestion
    "description": "A military flight helmet. Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitted with plastic rails to allow for different load-outs.",
```

File: data/json/items/tool_armor.json [hunk 6]
```diff
@@ -3152,5 +3152,110 @@
...
+      }
+    ]
+  },
+  {
+    "id": "flight_helmet_on",
+    "repairs_like": "nomex_hood",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "military flight helmet" },
+    "description": "A military flight helmet. Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitted with plastic rails to allow for different load-outs.",
```
Comment (BrettDong) [inline data/json/items/tool_armor.json:3217]:
```suggestion
    "description": "A military flight helmet.  Adorned with a visor, microphone and a noise dampening system.  The helmet is also fitted with plastic rails to allow for different load-outs.",
```

**Comment by Pericles1848** [comment]
This is pretty cool. I think it would be interesting to add a pair of army aviator wings that spawn on the zombie military pilots and are similar to the police badges with zero encumbrance, but that would likely be a separate PR. I'd love to see this stuff in game!

**Comment by faefux** [comment]
> This is pretty cool. I think it would be interesting to add a pair of army aviator wings that spawn on the zombie military pilots and are similar to the police badges with zero encumbrance, but that would likely be a separate PR. I'd love to see this stuff in game!

Oh, that's a neat idea, thanks! Just googled and I did not know they existed. I'll add them in to this PR. I think i'll add two types, gold and silver and have ones for the pre catacylsm army and the new Old Guard.

**Comment by Pericles1848** [comment]
I've been looking into the various aircraft in CDDA and who operates them and thus what sort of wings would likely be present, (as I have a hunch the Marines operate the V-22) I'll probably make a PR when I'm done looking into it. I'm pretty sure the old guard has a large stockpile of current wings and rate badges (they seem to stock marshal badges at least) but I imagine there is a very sudden need for blob specialists, weapon techs for those "exodii" weapons, and many more things that really weren't a concern to DoD before the Cataclysm, so there are likely new badges/rates that are in existence.

**Comment by Tairesh** [comment]
Are you planning to add this suit and helmet to the Military Pilot profession?

**Comment by faefux** [comment]
> Are you planning to add this suit and helmet to the Military Pilot profession?

I was not sure if such a change would be welcome. Would you like me to add to the starting profession and pilot zombie drops, @tairesh ?

**Comment by Tairesh** [comment]
> > Are you planning to add this suit and helmet to the Military Pilot profession?
> 
> I was not sure if such a change would be welcome. Would you like me to add to the starting profession and pilot zombie drops, @Tairesh ?

I think it would be awesome. Now a military pilot starts off in exactly the same clothes as a rifleman (without an ESAPI vest and MOLLE pack), a special suit and helmet would make a lot of sense.

**Pull request merged**